src/net/rpc.rs

```diff
@@ -612,6 +612,10 @@ impl ConversationHttp {
         peerdb: &PeerDB,
         handler_args: &RPCHandlerArgs,
     ) -> Result<(), net_error> {
+        // let timer = HTTP_REQ_HISTOGRAM
+        //     .with_label_values(&["/v2/info"])
+        //     .start_timer(); //promserver
+        monitoring::increment_rpc_request_counter("/v2/info".to_string(), "get".to_string()); //promserver
```

Comment:
I don't know what's the right way to resolve this, but we've been implementing the same thing, with 2 different approaches - https://github.com/blockstack/stacks-blockchain/pull/2453/commits/134671da798a61cf513672711c0f2ea8fb24fe7a.